src/components/Layout/ColorModeSwitcher.tsx

```diff
@@ -27,15 +27,16 @@ export const ColorModeSwitcher: React.FC<ColorModeSwitcherProps> = (props) => {
   )
 }
 
-export const DropdownColorModeSwitcher = (props) => {
+export const DropdownColorModeSwitcher = ({ ...props }) => {
+  const { t } = useTranslation()
   const { toggleColorMode } = useColorMode()
   const isLightMode = useColorModeValue(true, false)
   const SwitchIcon = useColorModeValue(IoMdMoon, IoMdSunny)
 
   return (
-    <MenuItem onClick={toggleColorMode} closeOnSelect={true} {...props}>
+    <MenuItem onClick={toggleColorMode} {...props}>
       <Icon as={SwitchIcon} />
-      {!isLightMode ? <Trans i18nKey='light_mode'>Light mode</Trans> : <Trans i18nKey='dark_mode'>Dark mode</Trans>}
+      <Text as='span'>{isLightMode ? t('dark_mode') : t('light_mode')}</Text>
```

Comment:
You removed the fallbacks here for a change that was only adding a wrapping Text component, not required at all.